Makefile.dep

```diff
@@ -12,6 +12,31 @@ ifneq (,$(filter nhdp,$(USEMODULE)))
   USEMODULE += oonf_rfc5444
 endif
 
+ifneq (,$(filter gnrc_netif_default,$(USEMODULE)))
+  USEMODULE += gnrc_netif
+  USEMODULE += netif_default
```

Comment:
Hm, okay, now back to your original proposal. If I do:
```
ifneq (,$(filter netif_default,$(USEMODULE)))
    USEMODULE += at86rf231
endif
```
in the board makefile like I originally had, who should pull in the `gnrc_netif_default` then?